Dockerfile

```diff
@@ -4,7 +4,8 @@ LABEL maintainer="Daniel Schröder <daniel.schroeder@skriptfabrik.com>"
 
 ARG ELEMENTS_CLI_VERSION=latest
 
-ENV HOSTNAME=0.0.0.0
+ENV LISTEN_ADDRESS=0.0.0.0
+ENV HOSTNAME=localhost
```

Comment:
```suggestion
ENV ELEMENTS_HOSTNAME=0.0.0.0
```

Reply:
So you want to also change line 22 to use `ELEMENTS_HOSTNAME`? Or `ELEMENTS_HOSTNAME` should override `HOSTNAME`?